src/methods/gas_price.rs

```diff
@@ -1,3 +1,57 @@
+//! Returns the gas price for a specific block_height or block_hash.
+//!
+//! ## Examples
+//!
+//! Returns the gas fees for this block:
+//! <https://explorer.near.org/blocks/6atGq4TUTZerVHU9qWoYfzXNBg3K4C4cca15TE6KfuBr>
```

Comment:
this hash...

Reply:
Documented both `hash` and `height`, hence the URL.

I can change it to what's below if it'll make it look better

> Returns the gas fees for this [block](https://explorer.near.org/blocks/6atGq4TUTZerVHU9qWoYfzXNBg3K4C4cca15TE6KfuBr)